content/pt-br/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,263 @@
+```
+Você então verá que esses contêineres de inicialização terminam, e que `myapp-pod` Pod vão para este estado de execução:
+
+```shell
+kubectl get -f myapp.yaml
+```
+A saída deverá ser similar a isso:
+```
+NAME        READY     STATUS    RESTARTS   AGE
+myapp-pod   1/1       Running   0          9m
+```
+Esse simples exemplo deve provê alguma inspiração para você criar seus próprios contêineres de inicialização. [O que vem por ai](#what-s-next)contém um link para mais exemplos detalhados.
+
+## Comportamento detalhado
+
+Durante a inicialização do Pod, o kuberlet espera a execução dos contêineres de inicialização até que a conexão e armazenamento estejam prontos. Então o kuberlet executa os Pod's contêineres de inicialização na ordem que eles aparecem na especificação dos Pod's. 
+
+Cada contêiner de inicialização deve sair com sucesso antes do próximo contêiner iniciar. Se um contêiner falhar para iniciar devido a tempo de execução ou sair com falha, ele tenta novamente de acordo com o Pod `restartPolicy`. Contudo, se a Pod `restartPolicy` está configurada para Always, os icontêineres de inicialização usam `restartPolicy` OnFailure.
+
+Um Pod não pode estar `Ready` até que todos os contêineres de inicialização tenham sucesso. A porta no contêiner de inicialização não são agregadas sob os Serviços. O Pod que está inicializando está no estado `Pending` mas deve ter a condição `Initialized` definida para falso.
+
+Se o Pod [reiniciar](#razões-do-pod-reiniciar), ou ser reiniciado, todos os contêineres de inicialização devem executar novamente.
+
+Mudanças na especificação do contêiner de inicialização são limitadas para o campo da imagem do contêiner.
+Alterando o campo da imagem do contêiner de inicialização é equivalente a reiniciar o Pod.
+
+Por conta do contêiner de inicialização poder ser reiniciado, tentado novamente, ou re-executado, código do contêiner de inicialização pode ser idempontente. Em particular, código que escreve em arquivos no `EmptyDirs` devem estar preparados para a possibilidade do arquivo de sáida já existir.
+
+Contêineres de inicialização tem todos os campos de um contêiner de aplicação. Contudo, Kubernets proibe `readinessProbe` de ser usado por conta que os contêineres de inicialização não consegue definir com prontidão distinta da conclusão.
+
+Use `activeDeadlineSeconds` no Pod para prevenir os contêineres de inicialização de falharem para sempre.
+A linha de conclusão ativa inclui os contêineres de inicialização.
+Contudo é recomendado usar `activeDeadlineSeconds` apenas se o time deploy sua aplicação como um Job, por conta de `activeDeadlineSeconds` ter efeito mesmo depois do contêiner de inicialização terminar.
+O Pod que está pronto e executando corretamente deve ser morto pelo `activeDeadlineSeconds` se você definir.
+O Nome de cada aplicação e contêiner de inicialização no Pod deve ser único; Um erro de validação é lançado por qualquer contêiner compartilhado com o mesmo nome.
+
+### Recursos
```

Comment:
This title should have a `{#resources}` at the end, so that the link up in the page still works. Example:
```
### Recursos {#resources}
```